openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -568,9 +568,15 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
             col = list(cols[0])
 
         # create subset name `familyTaskSubset_AOV`
-        group_name = 'render{}{}{}{}'.format(
-            task[0].upper(), task[1:],
-            subset[0].upper(), subset[1:])
+        # TODO refactor/remove me
+        family = skeleton["family"]
+        if not subset.startswith(family):
+            group_name = '{}{}{}{}{}'.format(
+                family,
+                task[0].upper(), task[1:],
+                subset[0].upper(), subset[1:])
+        else:
+            group_name = subset
```

Comment:
Not sure about this logic change. I mean, admittedly, it's weird that originally this was hardcoded to `render` prefix. However, the new logic doesn't appear much more logical to me either. The fact that subset starts with family, isn't that a template rule in settings too?

Reply:
I think that the whole part with concatenating group_name should be removed altogether and depend only on setting subset_name in the creator. I assume that this logic expected subset_name to be `Main` only which is a bit weird.